CHANGELOG.md

```diff
@@ -3,6 +3,68 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.18.8
+
+_Jan 11, 2024_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- ⏰ Support date-fns v3 (#11659) @LukasTy
+  Pickers support both v2 and v3 of date-fns. For v3 use `AdapterDateFnsV3`.
+  ```js
+  // with date-fns v2.x
+  import { AdapterDateFns } from '@mui/x-date- 
+  pickers/AdapterDateFns';
+  import de from 'date-fns/locale/de';
+  ```
+  ```js
+  // with date-fns v3.x
+  import { AdapterDateFns } from '@mui/x-date- 
+  pickers/AdapterDateFnsV3';
+  import { de } from 'date-fns/locale/de';
+  ```
+
+### Data Grid
+
+#### `@mui/x-data-grid@# 6.18.8`
+
+- [l10n] Add missing Danish (da-DK) locale export (#11641) @etlos
+
+#### `@mui/x-data-grid-pro@# 6.18.8` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@# 6.18.8`.
+
+#### `@mui/x-data-grid-premium@# 6.18.8` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@# 6.18.8`.
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@# 6.18.8`
+
+- [pickers] Add date-fns@3.x adapter (#11659) @LukasTy
```

Comment:
Technically this would warrant a minor release, because there is a new API. 🤔 
WDYT, should we do it? 🤔

Reply:
Yes, make sens